layouts/shortcodes/kf-deployment-ui-version.html

```diff
@@ -1 +1 @@
-v0.6.2
\ No newline at end of file
+v0.7.0
```

Comment:
Remove the new line at end of file.

We're using Hugo shortcodes as inline variables, and a new line in the shortcode breaks the formatting. For example, see the preview generated for your PR: https://deploy-preview-1458--competent-brattain-de2d6d.netlify.com/docs/gke/deploy/deploy-ui/